egui/src/widgets/plot/mod.rs

```diff
@@ -208,11 +213,43 @@ impl Plot {
     /// .show(ui, |plot_ui| plot_ui.line(line));
     /// # });
     /// ```
-    pub fn custom_label_func<F: 'static + Fn(&str, &Value) -> String>(
+    pub fn hover_label_func<F: 'static + Fn(&HoverConfig, &str, &Value) -> String>(
         mut self,
-        custom_lebel_func: F,
+        hover_label_func: F,
     ) -> Self {
-        self.custom_label_func = Some(Box::new(custom_lebel_func));
+        self.hover_label_func = Box::new(hover_label_func);
+        self
+    }
+
+    pub fn default_hover_label_func() -> Box<dyn Fn(&HoverConfig, &str, &Value) -> String> {
+        Box::new(|config, name, value| {
+            let mut prefix = String::new();
+
+            if !name.is_empty() {
+                prefix = format!("{}\n", name);
+            }
+
+            let x_decimals = 6 - ((value.x.abs().log10()).ceil().at_least(0.0) as usize).at_most(6);
+            let y_decimals = 6 - ((value.y.abs().log10()).ceil().at_least(0.0) as usize).at_most(6);
```

Comment:
This seems to be used in a few places, also in `bar.rs` and `box_elem.rs` -- maybe extract it to a function?
A few similar helper functions are already at the end of `items/mod.rs`.